app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,7 @@
  */
 @import "bootstrap-sprockets";
 @import "bootstrap";
+
+.discount-price {
+  color: #FF0000;
```

Comment:
Color literals like `#FF0000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF0000` should be written as `#ff0000`